### PR TITLE
adding optional logoLink param + updating the readme because I got stuck

### PR DESCRIPTION
**Pull request by JennaBarbara**

### CODE REVIEW

File: projects/ddap-common-lib/src/lib/layout/menu/menu.component.html [hunk 1]
```diff
@@ -4,13 +4,26 @@
...
-    <a class="site-name" routerLink="/">{{siteName}}</a>
+    <ng-container *ngIf="logoLink; else noLogoLink">
+      <a href="{{logoLink}}">
+        <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
+      </a>
+      <span *ngIf="isSandbox"
+        class="sandbox-warning"
+        matTooltip="This is sandbox testing environment">
+        Sandbox environment
+      </span>
```
Comment (shiroyuki) [inline projects/ddap-common-lib/src/lib/layout/menu/menu.component.html]: I don't like the code duplicate here. I think this block should come before the logo or after the site name so that we don't need an extra copy? What do you think, @monicavalluri, @mpanik?

File: projects/ddap-common-lib/src/lib/layout/menu/menu.component.html [hunk 2]
```diff
@@ -4,13 +4,26 @@
...
-    <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
-    <span *ngIf="isSandbox"
-      class="sandbox-warning"
-      matTooltip="This is sandbox testing environment">
-      Sandbox environment
-    </span>
-    <a class="site-name" routerLink="/">{{siteName}}</a>
+    <ng-container *ngIf="logoLink; else noLogoLink">
+      <a href="{{logoLink}}">
+        <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
```
Comment (shiroyuki) [inline projects/ddap-common-lib/src/lib/layout/menu/menu.component.html]: @monicavalluri @mpanik What do you think if we have the logo and the site name in the same anchor tag?

**Review by shiroyuki — COMMENTED** [review]
Functionality-wise, this looks good. However, the code is a bit hard to follow and I would like more input from everyone on code refactoring to improve code quality here.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@
 
 One-line command: `npm run prepublish && npm link ./dist/ddap-common-lib/`.
 
+Run the following in any project you are using with the library to test changes locally: `npm link ddap-common-lib`.
```
Comment (mpanik) [inline README.md:11]: +1 for keeping it here as well, but just note the information is also included in every ddap repo (see for example https://github.com/DNAstack/ddap-explore#working-with-ddap-library)

File: projects/ddap-common-lib/src/lib/layout/menu/menu.component.html (hunk 2 above)
Comment (mpanik) [reply projects/ddap-common-lib/src/lib/layout/menu/menu.component.html]: @shiroyuki sgtm

File: projects/ddap-common-lib/src/lib/layout/menu/menu.component.html (hunk 1 above)
Comment (mpanik) [reply projects/ddap-common-lib/src/lib/layout/menu/menu.component.html]: +1 the duplication is definitely not necessary here. @JennaBarbara try inlining the condition and removing the ng-template. Let me know if you need help.

File: projects/ddap-common-lib/src/lib/layout/menu/menu.component.ts [hunk 4]
```diff
@@ -18,6 +18,9 @@ export class MenuComponent {
   isSandbox = false;
   @Input()
   logoUrl: string;
+  @Input()
+  logoLink?: string;
```
Comment (mpanik) [inline projects/ddap-common-lib/src/lib/layout/menu/menu.component.ts]: I feel like those two properties are very similar in meanings and will lead to confusion. If you are going to anchor both elements under single `<a>` tag you can probably rename it to something more abstract or at least add comment. Perhaps: `headerLogoLink`

**Review by mpanik — CHANGES_REQUESTED** [review]
